package.json

```diff
@@ -63,9 +63,11 @@
     "md5": "^2.2.1",
     "moment": "^2.24.0",
     "nconf": "^0.10.0",
+    "node-cache": "^5.1.2",
     "node-fetch": "^2.6.1",
     "nodemailer": "^6.3.0",
     "nunjucks": "^3.2.0",
+    "prettier": "^2.1.2",
```

Comment:
@leomrocha having prettier in frontend-v2 might break translations, I think we need to add everthing that's in `i18n` folders (one should be in [frontend-v2 root](https://github.com/datopian/frontend-v2/tree/master/i18n), others might be in theme folders) to `.prettierignore`

Reply:
Fixing it